Integrations/Gmail/Gmail.py

```diff
@@ -567,7 +579,39 @@ def filters_to_entry(title, mailbox, response):
         'Contents': context,
         'ReadableContentsFormat': formats['markdown'],
         'HumanReadable': tableToMarkdown(title, context, headers, removeNull=True),
-        'EntryContext': {'GmailFilter(val.ID && val.ID == obj.ID)': context}
+        'EntryContext': {'GmailFilter(val.ID && val.ID == obj.ID)': context,
+                         'Gmail.Filter(val.ID && val.ID == obj.ID)': context}
```

Comment:
not sure what you wanna do here..
if you think `GmailFilter` is bad, then remove it from the yml, otherwise, keep as it was

Reply:
I saw that it should be Gmail.Filter and in order to not brake BC i've just added the right correction.

Reply:
thats ok, but lets remove `GmailFilter` from the yml